resources/js/Pages/Admin/SchoolsPanel.vue

```diff
@@ -85,7 +84,7 @@ function startFetching() {
           autocomplete="off"
           :disabled="!editing"
           placeholder=""
-          class="size-full xs:h-8 bg-transparent outline-none font-bold text-lg xs:mb-0"
+          class="size-full xs:h-8 bg-transparent outline-none font-bold text-lg xs:mb-0 resize-none"
```

Comment:
I see that you decided to disable the ability to resize the input with `resize-none`. Maybe it is a good idea to add  `input.style.resize = 'none'` line `vDynamicTextAreaHeight.ts` > `initDynamicHeightCalc` instead? That way this style doesn't neet to be manually added every time someone uses `vDynamicTextAreaHeight`.

Reply:
Great idea, implemented.